browser/plugins/three_scene.plugin.js

```diff
@@ -66,6 +68,25 @@
 	}
 
 	ThreeScenePlugin.prototype.update_input = function (slot, data) {
+		var that = this
+		var i
+
+		this.clickableObjectsInSlot[slot.index] = 0
+
+		function addItem(item) {
+			item.parent = parent
+
+			item.traverse(function(child) {
```

Comment:
this might cause a noticeable delay (and might be that it is updated every frame if the objects have any dynamically changing properties) which would kill framerate.